src/main/java/com/mixpanel/android/surveys/SurveyActivity.java

```diff
@@ -220,6 +183,8 @@ public void onClick(View v) {
 
         final Animation fadeIn = AnimationUtils.loadAnimation(this, R.anim.com_mixpanel_android_fade_in);
         closeButtonWrapper.startAnimation(fadeIn);
+
+        if (inApp.getStyle().equals("light")) showLightStyle();
```

Comment:
Can we do this instead?
```
if (inApp.getStyle().equals("light")) {
    showLightStyle();
}
```

Also, it'd be super useful if we have an `ENUM` instead of hardcoding the string.